components/base/Card.tsx

```diff
@@ -88,27 +88,29 @@ type Props = {
   description: string;
 };
 
-function Card(props: Props) {
+function Card({ description, badge, centerAlign, icon, link, topic }: Props) {
```

Comment:
This is similar to what you had before I think. (I just realized you can do this with TypeScript, woot!)